### PR TITLE
UI Improvements

### PR DESCRIPTION
**Pull request by ShayDavidson**
- ActionBar theme.
- Better event list.
- AND STUFF!

event_list

### CODE REVIEW

File: app/app.iml [hunk 1]
```diff
@@ -90,6 +90,7 @@
     <orderEntry type="library" exported="" name="parceler-api-0.2.16-SNAPSHOT" level="project" />
     <orderEntry type="library" exported="" name="support-annotations-21.0.3" level="project" />
     <orderEntry type="library" exported="" name="activeandroid-3.1.0-SNAPSHOT" level="project" />
+    <orderEntry type="library" exported="" name="picasso-2.5.0" level="project" />
```
Comment (ShayDavidson) [inline app/app.iml:93]: gives a simple api to background loading of images + handles re-use in list views (see `EventsAdapter` for usage).

File: app/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -2,6 +2,8 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="com.paypal.psix" >
 
+    <uses-permission android:name="android.permission.INTERNET" />
```
Comment (ShayDavidson) [inline app/src/main/AndroidManifest.xml:5]: required in order to load the images

File: app/src/main/res/values/styles.xml [hunk 3]
```diff
@@ -7,8 +7,14 @@
...
 
-    <style name="PSixTheme" parent="@style/Theme.AppCompat">
+    <style name="PSixTheme" parent="@style/Theme.AppCompat.Light.DarkActionBar">
         <item name="android:windowAnimationStyle">@style/SlideActivityAnimation</item>
+        <item name="actionBarStyle">@style/CustomBarStyle</item>
+    </style>
+
+    <style name="CustomBarStyle"
+        parent="@style/Widget.AppCompat.Light.ActionBar.Solid.Inverse">
+        <item name="background">@color/main_color</item>
```
Comment (ShayDavidson) [inline app/src/main/res/values/styles.xml:17]: about 2 hours of work to make the actionbar color to work :|

File: app/app.iml (hunk 1 above)
Comment (avihut) [inline app/app.iml:93]: Picasso looks awesome!

File: app/src/main/java/com/paypal/psix/fragments/EventsFragment.java [hunk 4]
```diff
@@ -70,4 +61,25 @@ public void onItemClick(AdapterView<?> adapterView, View view, int position, lon
...
     }
+
+    private ArrayList<Event> eventsDataSource() {
+        // Demo data.
+        Event[] array = {
+                Event.GenerateRandomEvent(), Event.GenerateRandomEvent(), Event.GenerateRandomEvent(),
+                Event.GenerateRandomEvent(), Event.GenerateRandomEvent(), Event.GenerateRandomEvent(),
+                Event.GenerateRandomEvent(), Event.GenerateRandomEvent(), Event.GenerateRandomEvent(),
+                Event.GenerateRandomEvent(), Event.GenerateRandomEvent(), Event.GenerateRandomEvent()
+        };
```
Comment (avihut) [inline app/src/main/java/com/paypal/psix/fragments/EventsFragment.java:72]: Not that it actually matters... but still, this hurts my eyes...

why not a for loop

**Comment by avihut** [comment]
Looks Osem!

File: app/src/main/java/com/paypal/psix/fragments/EventsFragment.java (hunk 4 above)
Comment (ShayDavidson) [inline app/src/main/java/com/paypal/psix/fragments/EventsFragment.java:72]: temp being temp :P

**Pull request merged**